pkg/controller/policyrecommendation/policyrecommendation_controller.go

```diff
@@ -439,6 +432,30 @@ func (r *ReconcilePolicyRecommendation) Reconcile(ctx context.Context, request r
 		return reconcile.Result{}, nil
 	}
 
+	policyRecommendationCfg := &render.PolicyRecommendationConfiguration{
+		ClusterDomain:                  r.clusterDomain,
+		Installation:                   installation,
+		ManagedCluster:                 isManagedCluster,
+		PullSecrets:                    pullSecrets,
+		Openshift:                      r.provider == operatorv1.ProviderOpenShift,
+		UsePSP:                         r.usePSP,
+		Namespace:                      helper.InstallNamespace(),
+		Tenant:                         tenant,
+		BindingNamespaces:              bindNamespaces,
+		ExternalElastic:                r.externalElastic,
+		TrustedBundle:                  trustedBundleRO,
+		PolicyRecommendationCertSecret: policyRecommendationKeyPair,
+	}
+
+	// Render the desired objects from the CRD and create or update them.
+	component := render.PolicyRecommendation(policyRecommendationCfg)
+
+	if err = imageset.ApplyImageSet(ctx, r.client, variant, component); err != nil {
+		r.status.SetDegraded(operatorv1.ResourceUpdateError, "Error with images from ImageSet", err, logc)
+		return reconcile.Result{}, err
+	}
+
+	components = append(components, component)
```

Comment:
By adding the component after the other components, doesn't the operator try to render tls assets before the namespace exists on the first ever iteration? I think that would create an unnecessary error in the logs / tigerastatus.